src/server/Document/index.jsx

```diff
@@ -23,7 +24,15 @@ const renderDocument = async ({
   url,
 }) => {
   const isDev = process.env.NODE_ENV === 'development';
-  const cache = createCache({ key: 'bbc' });
+  const cache = createCache({
+    key: 'bbc',
+    stylisPlugins: [
+      element => {
+        element.value = element.value.replaceAll('0.', '.');
+      },
```

Comment:
If we think we should add this, this will be tidier in a new separate directory for pre-processor plugins